src/styles/NavbarStyles/index.less

```diff
@@ -1,123 +1,188 @@
-  font-size: 2.2rem;
-  letter-spacing: 1px;
-  align-items: center;
-  font-weight: 500;
-  color: rgb(58, 9, 42);
-  font-family: 'Staatliches', 'cursive';
+  font-family: @text__font-family;
   overflow: hidden;
-  white-space: nowrap;
-}
+  transition: 0.1s all ease-out;
 
-.navOptions {
-  display: flex;
-  justify-content: space-evenly;
-  align-items: center;
-  width: 40%;
-  margin-right: 2rem;
-}
-.navLinks {
-  color: rgb(58, 9, 42);
-  font-weight: bold;
-  font-size: 1.1rem;
-}
-.navLinks:hover {
-  color: #fff;
-  text-decoration: underline;
-  text-decoration-thickness: 10px;
-}
+  .navbar__logo {
+    display: flex;
+    align-items: center;
+    width: 22%;
 
-.navButtons {
-  width: 20%;
-  display: flex;
-  justify-content: space-evenly;
-  align-items: center;
-}
+    .navbar__navTitle {
+      margin: 0 0 0 1.5rem;
+      font-size: 1.95rem;
+      letter-spacing: 1px;
+      align-items: center;
+      color: #3b0e2b;
+      font-family: @text__font-family--header;
+      overflow: hidden;
+      white-space: nowrap;
+      transition: 0.2s all ease-in;
+    }
+    @media screen and (max-width: 1220px) {
+      .navbar__navTitle {
+        margin: 0;
+      }
+    }
+  }
 
-.navSocial {
-  display: flex;
-  justify-content: space-evenly;
-  align-items: center;
-  width: 10%;
-  height: auto;
-  margin-right: 4rem;
-}
+  .navbar__links {
+    display: flex;
+    width: 40%;
+    justify-content: space-evenly;
+    margin-left: 0.6rem;
 
-.socialIcon {
-  font-size: 2rem;
-  margin: 0.7rem;
-  color: rgb(58, 9, 42);
-}
+    .navbar__navLink {
+      color: #3b0e2b;
+      font-size: 0.95rem;
+      font-weight: 600;
+      transition: 0.1s all ease-in;
+    }
+    .navbar__navLink:hover {
+      color: #fff;
+      text-decoration: underline;
+      text-decoration-thickness: 10px;
+    }
+    @media screen and (max-width: 1220px) {
+      .navbar__navLink {
+        font-size: 0.85rem;
+      }
+    }
+  }
+  @media screen and (max-width: 1070px) {
+    .navbar__links {
+      width: 50%;
+      justify-content: space-between;
+    }
+  }
 
-.socialIcon:hover {
-  color: white;
-}
+  .navbar__socials {
+    display: flex;
+    width: 15%;
+    justify-content: space-evenly;
+    transition: 0.2s all ease-in;
 
-.navContact {
-  width: 70%;
-  font-size: 1rem;
-}
+    .navbar__socials__cont {
+      display: flex;
+      width: 100%;
+      justify-content: space-evenly;
+    }
 
-.navSignup {
-  width: 55%;
-  font-size: 1rem;
-}
+    .navbar__socialLink {
+      font-size: 1.7rem;
+      color: #3b0e2b;
+      transition: 0.1s all ease-in;
+    }
+    .navbar__socialLink:hover {
+      color: white;
+    }
+    @media screen and (max-width: 1100px) {
+      .navbar__socialLink {
+        margin-right: 0;
+        font-size: 1.5rem;
+      }
+    }
+  }
+  @media screen and (max-width: 1070px) {
+    .navbar__socials {
+      display: none;
+    }
+  }
 
-.navSignup__hide {
-  display: none;
-}
+  .navbar__btns {
```

Comment:
i like this way of styling better than normal CSS styling. I am definitely following your footsteps when it comes to styling. I've review the way you did your styling on the LandingInstructor page and it looks very organized and clean. Keep it up!